packages/expo-notifications/android/src/main/java/expo/modules/notifications/userinteractionreceiver/UserInteractionReceiver.java

```diff
@@ -0,0 +1,75 @@
+package expo.modules.notifications.userinteractionreceiver;
+
+import android.app.RemoteInput;
+import android.content.Context;
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v4.app.NotificationManagerCompat;
+
+import expo.modules.notifications.action.NotificationActionCenter;
+import expo.modules.notifications.NotificationConstants;
+import expo.modules.notifications.postoffice.PostOfficeProxy;
+
+import static expo.modules.notifications.NotificationConstants.NOTIFICATION_ACTION_TYPE_KEY;
+import static expo.modules.notifications.NotificationConstants.NOTIFICATION_APP_ID_KEY;
+import static expo.modules.notifications.NotificationConstants.NOTIFICATION_OBJECT_KEY;
+import static expo.modules.notifications.NotificationConstants.NOTIFICATION_STICKY;
+
+public class UserInteractionReceiver {
```

Comment:
Is this class used anywhere?

Reply:
This class is responsible for intercepting info about user interaction from intent. (e.g. user tapped on the notification). It's used in 'onNewIntent()' and 'onCreate()' methods.